tensorboard_logger/tensorboard_logger.py

```diff
@@ -82,7 +82,10 @@ def _make_tf_name(self, name):
 
     def _log_value(self, tf_name, value, step=None):
         summary = summary_pb2.Summary()
-        summary.value.add(tag=tf_name, simple_value=value)
+        if type(value) is summary_pb2.Summary.Image:
```

Comment:
I think ``isinstance(value, summary_pb2.Summary.Image)`` would be more robust, unless we specifically want to exclude any possible subclasses.